### DISCUSSION TITLE
How do I use onClick with next/image (<Image>)?

### DISCUSSION THREAD

**Discussion by opqpop** [vercel/next.js#28387] [Help] [answered]
Hello, I'd like to run some javascript rather than navigate to another page upon clicking a next/image, something like this:

```
import Image from 'next/image'

const Bob = () => {
  return <Image src={'path'} onClick={() => console.log('hi')}/>
}
```

But the onClick doesn't seem to work. Is there a way to do this? Couldn't quite figure it out from the documentation here: https://nextjs.org/docs/api-reference/next/image

Thanks!

**Comment by jamesmosier** [comment, marked as answer]
The `Image component does not accept an `onClick` prop, but you could wrap the `<Image />` element in a `<div>` or `<a>` tag and place the `onClick` handler on that element.

**Reply by opqpop** [reply]
Oh I see, that did the trick! Thanks so much!

**Reply by spenceryonce** [reply]
What's an or tag?

**Reply by conor909** [reply]
Do you loose all SSR & SEO value on anything that's clickable?
